src/Tribe/Tickets.php

```diff
@@ -513,12 +514,21 @@ public function has_permission( $post, $data, $nonce_action ) {
 				do_action( 'tribe_tickets_ticket_added', $post_id );
 			}
 
-			$return = array( 'html' => $return );
+			parse_str ( $_POST['formdata'], $post_data );
+
+			$ticket = $this->get_ticket( $post_id, $ticket_id );
+			$post_data['ticket_stock'] = $ticket->stock;
+			$post_data['ticket_capacity'] = $ticket->original_stock();
+
+			$return = array(
+				'html' => $return,
```

Comment:
Maybe I'm missing it ... but with all the restructuring in this method I don't think `$return` is actually defined by this point, so we have nothing to assign to the `'html'` key